src/common/types/environment-variables.ts

```diff
@@ -66,7 +66,7 @@ export class EnvironmentVariables {
       || defaultValues.pegoutMinValue;
     this.pegoutMaxValue = process.env.VUE_APP_PEGOUT_MAX_AMOUNT_ALLOWED_IN_RBTC
       || defaultValues.pegoutMaxValue;
-    this.minFeePerKb = {
+    this.minFeeSatPerByte = {
       fast: process.env.VUE_APP_FEE_PER_KB_FAST_MIN
```

Comment:
Would be nice to change these variable and default value names to avoid future confusion.